udata/core/organization/models.py

```diff
@@ -235,6 +243,49 @@ def json_ld(self):
 
         return result
 
+    @property
+    def members_count(self):
+        return len(self.members)
+    
+    @property
+    def views_count(self):
+        return self.metrics['views']
+
+    def count_datasets(self):
+        from udata.models import Dataset
+        self.metrics["datasets"] = Dataset.objects(organization=self).visible().count()
+        self.save()
+
+    def count_reuses(self):
+        from udata.models import Reuse
+        self.metrics["reuses"] = Reuse.objects(organization=self).count()
+        self.save()
+
+    def count_followers(self):
+        from udata.models import Follow
+        self.metrics["followers"] = Follow.objects.followers(self).count()
+        self.save()
+    
+    @classmethod
+    def get_metrics_keys(cls):
+        return [
+            "datasets",
+            "members",
+            "reuses",
+            "followers",
+            "views",
+        ]
+
+    @property
+    def get_metrics(self):
```

Comment:
use `get_metrics_keys()` here

Reply:
well, not really possible here 🤔 this pattern (keys duplication) is broken. we should be able to find a better way 🌈